Code/Conviva/Source/Events/ConvivaReporters/AppEventConvivaReporter.swift

```diff
@@ -44,24 +46,36 @@ class AppEventConvivaReporter: AppEventProcessor {
         self.analytics.reportAppBackgrounded()
     }
     
-    func appGotNewAccessLogEntry(event: AVPlayerItemAccessLogEvent, isPlayingAd: Bool) {
-        // If the ad did load but we haven't started playing it,
-        // we will report the bitrate to the ad endpoint regardless.
-        // Previously, we were incorrectly reporting to the video endpoint
-        // when the ad loaded but was not playing.
-        let toSendToAdEndpoint = prerollAdPlaying || isPlayingAd
-        let endpoint = toSendToAdEndpoint ? self.adAnalytics : self.videoAnalytics
+    func appGotNewAccessLogEntry(event: AVPlayerItemAccessLogEvent, item: AVPlayerItem, isPlayingAd: Bool) {
         
+        // if we get the same bitrate for the same item in a row, we don't need to report that.
+        // I see this happens when we get an access log towards the end of an ad.
+        // as well, which might be indicating something else other than a bitrate change.
+        if lastPlayerItem == item, event.indicatedBitrate == lastAccessLogEvent?.indicatedBitrate {
```

Comment:
Wouldn't this early return now block the reporting of dropped video frames?
I can see scenarios where the `lastPlayerItem == item` and `event.indicatedBitrate == lastAccessLogEvent?.indicatedBitrate` are true but `numberOfDroppedVideoFrames >= 0` is also true. In that case we still want to report the `CIS_SSDK_PLAYBACK_METRIC_DROPPED_FRAMES_TOTAL` metric.